sensor_msgs/PointCloud2 cloud_points_with_normals

nav_msgs/OccupancyGrid ground_level_map

nav_msgs/OccupancyGrid height_map
float32 height_map_scale                # scaling factor of values in height map
